user-interface/src/lib/components/uswds/Alert.tsx

```diff
@@ -95,12 +95,12 @@ function AlertComponent(props: AlertProps, ref: React.Ref<AlertRefType>) {
       >
         <div className="usa-alert__body">
           {props.title && <h4 className="usa-alert__heading">{props.title}</h4>}
-          <p
+          <div
             className="usa-alert__text"
             data-testid={`alert-message${props.id ? '-' + props.id : ''}`}
           >
             {props.message}
-          </p>
+          </div>
```

Comment:
This came up as an interesting warning in the code, because I switched from passing message as a simple string to actually allowing React Nodes to be passed.  It is invalid for an HTML paragraph to contain other HTML elements (with the exception of basic text formatting, spans etc.)